assets/src/components/common.tsx

```diff
@@ -133,4 +134,5 @@ export {
   Tag,
   Tooltip,
   Upload,
+  Tabs,
```

Comment:
nit - can we make sure the import above and this export are listed alphabetically?

Reply:
done